semantic_router/index/pinecone.py

```diff
@@ -127,19 +137,25 @@ def __init__(
         region: str = "us-east-1",
         host: str = "",
         namespace: Optional[str] = "",
-        base_url: Optional[str] = "https://api.pinecone.io",
+        base_url: Optional[str] = "",
```

Comment:
@theanupllm wouldn't this break `PineconeIndex` if used normally (ie not using pinecone local)?